scripts/fetchKakarotTx.ts

```diff
@@ -139,20 +182,44 @@ function transformTransactionsAndEvents(
     });
   });
 
-  return { transformedTransactions, eventsWithTransaction };
+  return {
+    transformedTransactions,
+    eventsWithTransaction,
+    toTypedEthTxTransaction,
+  };
 }
+
 async function main() {
   try {
     const targetCount = 100;
-    const transactions = await collectTransactions(targetCount);
-
+    const {
+      headersList,
+      eventsList,
+      transactionsList,
+      expectedTransform,
+      expectedToTypedEthTxTransactions,
+    } = await collectTransactions(targetCount);
+
+    // Save the collected transactions to a file
     await Deno.writeTextFile(
       "indexer/src/test-data/transactionsData.json",
```

Comment:
I saw that these paths are often written hardcoded in the code, maybe we could extract constants from them instead so we only have to write them once and easily replace them if needed, wdyt?